src/i32x4_.rs

```diff
@@ -490,6 +490,45 @@ impl i32x4 {
       }
     }
   }
+
+  #[inline]
+  #[must_use]
+  pub fn mul_widen(self, rhs: Self) -> i64x4 {
```

Comment:
could we put a doc comment here about what happens, like the others have.

Reply:
Hi there, let me know if there is anything else in the PR. Thanks!